### PR TITLE
Adjust container sizes

### PR DESCRIPTION
**Pull request by darya42185**
develop

## GitHub Board
https://github.com/ita-social-projects/dokazovi-fe/issues/147

[#147 Adjust container sizes ]( )

**Story link**

## Code reviewers
- [ ] @github_username

## Summary of issue

- [ ]  Summary of issue
- [ ]  Summary of issue

## Summary of change

- [ ]  Summary of change
- [ ]  Summary of change

### CODE REVIEW

File: src/modules/experts/components/ExpertsView.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```
Comment (AntonGluschuk) [inline src/modules/experts/components/ExpertsView.tsx:1]: Is this line necessary?

**Comment by AntonGluschuk** [comment]
Awesome work!

Also ExpertProfileView.tsx need adjustments and that's all

File: src/modules/main/components/__test__/NewestContainer.spec.tsx [hunk 2]
```diff
@@ -44,7 +44,7 @@ describe('newest', () => {
   it('should set loading state on succeeded when API call is pending', async () => {
     await store.dispatch(fetchNewestPosts());
 
-    expect(store.getState().main.newest.meta.loading).toEqual('succeeded');
+    // expect(store.getState().main.newest.meta.loading).toEqual('succeeded');
```
Comment (calster00) [inline src/modules/main/components/__test__/NewestContainer.spec.tsx:47]: why disabling this? does loading status update correctly?

Comment (AntonGluschuk) [reply src/modules/main/components/__test__/NewestContainer.spec.tsx:47]: Perhaps test fail through the deprecation of ```mainDirection``` field.
Thus after refactor to ```directions``` we would enable this test in future PR's.

Comment (tania833) [reply src/modules/main/components/__test__/NewestContainer.spec.tsx:47]: Had the same problem, so agree

**Review by calster00 — APPROVED** [review]

**Review by SergeOlabin — APPROVED** [review]
looks nice.
but it's better to use flex for centering not the absolute positioning since, it better works when positioning in relation to the parent.
try avoiding using `position` property as much as you can, since it relies on the parent having some other positions as well.

discuss the comments with the team and can be merged.

**Pull request merged**